net/restic-rest-server/Makefile

```diff
@@ -0,0 +1,50 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=restic-rest-server
+PKG_VERSION:=0.9.7
+PKG_RELEASE:=1
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/rest-server-$(PKG_VERSION)
+PKG_SOURCE:=rest-server-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/restic/rest-server/tar.gz/v${PKG_VERSION}?
+PKG_HASH:=922811cfc49879f7086fe559d25ec1bb4169b2c4a5df3559b499e6292f435661
+
+PKG_LICENSE:=BSD-2-Clause
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Markus Weippert <markus@gekmihesg.de>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/restic/rest-server/
+GO_PKG_BUILD_PKG:=github.com/restic/rest-server/cmd/rest-server/
+
```

Comment:
Not sure if the version number is embedded in the binary. If it isn't, I think you can add this:

```
GO_PKG_LDFLAGS_X:=main.version=$(PKG_VERSION)
```

You can also add

```
GO_PKG_LDFLAGS:=-s -w
```

to mirror the upstream build process, though OpenWrt binaries are usually stripped anyway which should accomplish the same thing as adding these flags.

Reply:
Version is automatically set from the VERSION file here: https://github.com/restic/rest-server/blob/master/build.go#L532
Flags added explicitly.

Reply:
The golang-package.mk build process doesn't use build.go. It calls `go install` directly.

build.go is used when [manually building from source](https://github.com/restic/rest-server#from-source):

```
go run build.go
```

Reply:
Sorry, you're right. The "version" was actually set to "manually". Fixed.